lib/dal/src/func/binding/action.rs

```diff
@@ -124,7 +124,7 @@ impl ActionBinding {
             action_kind,
             func.name.to_owned(),
             func.description.to_owned(),
-            schema_variant_id,
+            ActionPrototypeParent::SchemaVariant(schema_variant_id),
```

Comment:
I think you can just pass schema_variant_id here--it at least ought to have a `From`